channels/packages/node-manager/develop/daemonset-storageos-node-manager.yaml

```diff
@@ -26,9 +26,27 @@ spec:
         - --config=node_manager_config.yaml
         command:
         - /node-manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
         image: controller:latest
         imagePullPolicy: IfNotPresent
+        livenessProbe:
+          httpGet:
+            path: /healthz
```

Comment:
The healthz endpoint on node manager runs `os.Exit(1)`
Is this intended?

Reply:
It has an initial delay and some failure threshold. On the other hand `/healthz` is a ping service, so i don't think this should ever happen, or if happen maybe the best thing to do to exit :D